### PR TITLE
Adapter package clean up

### PR DESCRIPTION
**Pull request by AmaliMatharaarachchi**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by Krishanx92 — DISMISSED** [review]

**Comment by Amila-Rukshan** [comment]
shall we not merge this until Sampath's changes are merged, otherwise he will get many conflicts

**Review by Amila-Rukshan — APPROVED** [review]

**Pull request merged**
